app/api_topologies.go

```diff
@@ -31,6 +139,52 @@ type topologyStats struct {
 	FilteredNodes      int `json:"filtered_nodes"`
 }
 
+type byName []APITopologyDesc
+
+func (a byName) Len() int           { return len(a) }
+func (a byName) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a byName) Less(i, j int) bool { return a[i].Name < a[j].Name }
+
+func (r *registry) add(ts ...APITopologyDesc) {
+	r.Lock()
+	defer r.Unlock()
+	for _, t := range ts {
+		t.URL = apiTopologyURL + t.id
+
+		if t.parent != "" {
+			parent := r.items[t.parent]
+			parent.SubTopologies = append(r.items[t.parent].SubTopologies, t)
+			sort.Sort(byName(parent.SubTopologies))
+			r.items[t.parent] = parent
+		}
+
+		r.items[t.id] = t
```

Comment:
Unfortunately not; sub topologies are still looked up by id (they are not nested in their URL).